static/js/util/validation_test.js

```diff
@@ -385,15 +385,17 @@ describe('Profile validation functions', () => {
       assertMaybeEquality(Just(20), validateYear("-20"));
     });
 
-    it('returns 1800 for 4-character years less than 1800', () => {
-      assertMaybeEquality(Just(1800), validateYear("1799"));
-      assertMaybeEquality(Just(1800), validateYear("1099"));
+    it('returns a minimum of 120 years ago', () => {
+      let cutoff = moment().subtract(120, 'years').year();
+      assertMaybeEquality(Just(cutoff), validateYear("1799"));
+      assertMaybeEquality(Just(cutoff), validateYear("1099"));
     });
 
-    it('returns 2100 for years >= 2100', () => {
-      assertMaybeEquality(Just(2100), validateYear("2100"));
-      assertMaybeEquality(Just(2100), validateYear("2300"));
-      assertMaybeEquality(Just(2100), validateYear("52300"));
+    it('returns a maximum of the current year', () => {
+      let now = moment().year();
+      assertMaybeEquality(Just(now), validateYear("2100"));
+      assertMaybeEquality(Just(now), validateYear("2300"));
+      assertMaybeEquality(Just(now), validateYear("52300"));
```

Comment:
these are redundant. i think this assertion alone would do the trick

`assertMaybeEquality(Just(now), moment().add(3, "years").year());`